packages/tc-web/src/components/tc-collapsible/tc-collapsible.tsx

```diff
@@ -0,0 +1,65 @@
+import { Component, h, Prop, State } from "@stencil/core";
+import { parseTags } from "../../util/util";
+
+@Component({
+  tag: "tc-collapsible",
+  styleUrl: "tc-collapsible.css",
+  shadow: true,
+})
+export class TcCollapsible {
+  @Prop() colTitle: string;
+  @Prop() description: string = "";
+  @Prop() tags: string = "[]";
+  @Prop() backgroundColor: string = "rgb(255, 255, 255)";
+
+  @State() showDetails: boolean = false;
+  toggleDetails = () => {
+    this.showDetails = !this.showDetails;
+  };
+
+  render() {
+    return (
+      <div
+        class={{
+          "px-3": true,
+          "py-2": true,
+          border: true,
+          "border-gray-200": true,
+          "border-solid": true,
+          shadow: true,
+          "md:p-4": true,
+          "md:px-6": true,
+          "font-sans": true,
+          "font-normal": true,
+          "mb-2": true,
+        }}
+        style={{ backgroundColor: this.backgroundColor }}
+      >
+        <div class="flex flex-col items-start justify-between md:flex-row">
+          <div>
+            <h4 class="!m-0 text-xl md:text-3xl font-semibold">
+              {this.colTitle}
+            </h4>
+            <div class="mt-2 text-lg">{this.description}</div>
```

Comment:
should we hide this if there is no description set? otherwise we have an empty space (mt-2)